share/wrf_timeseries.F

```diff
@@ -342,8 +355,14 @@ SUBROUTINE calc_ts( grid )
             !
 #if (EM_CORE == 1)
             DO k=1,grid%max_ts_level
-            earth_u_profile(k) = grid%u_2(ix,k,iy)*grid%cosa(ix,iy)-grid%v_2(ix,k,iy)*grid%sina(ix,iy)
-            earth_v_profile(k) = grid%v_2(ix,k,iy)*grid%cosa(ix,iy)+grid%u_2(ix,k,iy)*grid%sina(ix,iy)
+                ! interpolation loop: if you want u,v on cell centers, tslist_interp = .true.
+                IF (config_flags%tslist_interp) THEN
+                    earth_u_profile(k) = (grid%u_2(ix,k,iy)+grid%u_2(ix+1,k,iy))/2.0 
+                    earth_v_profile(k) = (grid%v_2(ix,k,iy)+grid%v_2(ix,k,iy+1))/2.0 
```

Comment:
One potential issue here is that, if `config_flags%tslist_interp` is `.true.`, we get *grid-relative* winds rather than *earth-relative* winds. A user may think they are just getting cell-centered winds that are still earth-relative, but what they're actually getting are cell-centered winds that are grid-relative. I would suggest this need to be reworked to provide a consistent basis for the wind fields, whether they are cell-centered or at their native locations.